plugins/nodes/src/js/components/NodesTable.js

```diff
@@ -34,9 +34,28 @@ var NodesTable = React.createClass({
     };
   },
 
+  renderRegion(_prop, node) {
+    const name = node.getRegionName();
```

Comment:
I wonder what we should do in the case where Region or Zone is not set 🤔  I believe the direction from @leemunroe is to show `--` for cases where values are not set in tables/configmaps.

Reply:
Yes, when something isn't configured lets be consistent with what we show. And ideally this is a variable that is easily changed across the UI. I believe @ahoskins may already have something like this for service configuration. Not sure if `--` or `—`. Both work. I'm more concerned about consistency and using the same pattern.

Reply:
For the framework configuration, I am showing the review screen using the `HashMapDisplay` component. I added a prop called `emptyValue` to this component. I use "\u2014" (&mdash;) as the empty value.

Reply:
As far as I understood the feature, this will always be set, and if the user did not set this value mesos is providing a default value. But to be sure I may add a &mdash; (`&mdash;`);